### PR TITLE
Use clang-sys bindings instead of ours.

### PR DESCRIPTION
**Pull request by emilio**
I plan to use a dynamically loaded clang-sys library to remove the llvm-stable
feature.

This is part of the work.

r? @fitzgen

### CODE REVIEW

**Comment by emilio** [comment]
cc @KyleMayes

File: libbindgen/Cargo.toml [hunk 1]
```diff
@@ -25,13 +25,13 @@ quasi_codegen = "0.26"
...
 [dependencies]
 cfg-if = "0.1.0"
-clang-sys = "0.11.1"
 lazy_static = "0.1.*"
 libc = "0.2"
 rustc-serialize = "0.3.19"
 syntex_syntax = "0.50"
 regex = "0.1"
 cexpr = "0.2"
+clang-sys = { version = "0.11.1", features = ["gte_clang_3_6", "gte_clang_3_7", "gte_clang_3_8", "gte_clang_3_9"] }
```
Comment (fitzgen) [inline libbindgen/Cargo.toml]: I think we should try and keep the depencies in order (although I do realize they weren't already).

Comment (emilio) [inline libbindgen/Cargo.toml]: Done

**Pull request merged**